lib/client.js

```diff
@@ -2785,6 +2786,42 @@ MatrixClient.prototype.register = function(username, password,
 };
 
 /**
+ * Requests an email verification token for the purposes of registration.
+ * This API proxies the Identity Server /validate/email/requestToken API,
+ * adding registration-specific behaviour. Specifically, an account with
+ * the given email address already exists, it will either send an email
+ * to the address informing them of this or return M_THREEPID_IN_USE
+ * (which one is up to the Home Server).
+ *
+ * requestEmailToken calls the equivalent API directly on the ID server,
+ * therefore bypassing the registration-specific logic.
+ *
+ * @param {string} email
+ * @param {string} clientSecret
+ * @param {string} sendAttempt
+ * @param {string} nextLink Optional
```

Comment:
This parameter is not documented on https://github.com/matrix-org/matrix-doc/pull/343/files - what does it do?